File: Java/SelectionSort.java
import java.util.Scanner;

public class SelectionSort{  
    public static int[] selectionSort(int[] arr, int n){

        for (int i = 0; i < n; i++)  
        {  
            int index = i;  
            for (int j = i + 1; j < n; j++){  
                if (arr[j] < arr[index]){  
                    index = j; 
                }  
            }  
            int menor = arr[index];   
            arr[index] = arr[i];  
            arr[i] = menor;  
        }

        return arr;  

    }  
       
    public static void main(String a[]){

        int[] arr1 = new int[1000];
        int n;

        Scanner in = new Scanner(System.in);

        n = in.nextInt();
          

        for(int i = 0; i < n; i++)
        {
            arr1[i] = in.nextInt();
        }

        arr1 = selectionSort(arr1, n);
         
        System.out.println("Hello HacktoberFest!");  
        for(int i = 0; i < n; i++){  
            System.out.print(arr1[i]+" ");  
        }  
    }  
}  

File: Kotlin/SelectionSort.kt
private fun selectionSort(arr: IntArray, n: Int): IntArray {
    for (i in 0 until n) {
        var index = i
        for (j in i + 1 until n) {
            if (arr[j] < arr[index]) {
                index = j
            }
        }
        val menor = arr[index]
        arr[index] = arr[i]
        arr[i] = menor
    }

    return arr
}

fun main() {
    print("Digite o tamanho do Array: ")
    val n: Int = readLine()!!.toInt()
    var arr1 = IntArray(n)

    for (i in 0 until n) {
        print("Digite o elemento do Array: ")
        arr1[i] = readLine()!!.toInt()
    }

    arr1 = selectionSort(arr1, n)

    println("Hello HacktoberFest!")
    for (i in 0 until n) {
        print(arr1[i].toString() + " ")
    }
}
